--- origsrc/dirac-0.10.0/Makefile.am	2008-05-06 04:35:51.000000000 -0500
+++ src/dirac-0.10.0/Makefile.am	2008-06-15 03:40:58.796875000 -0500
@@ -1,5 +1,6 @@
 # $Id: dirac-0.10.0-1.src.patch,v 1.2 2008-06-15 08:48:18 yselkowitz Exp $
 #
+ACLOCAL_AMFLAGS = -I m4
 
 SUBDIRS = libdirac_byteio libdirac_common libdirac_motionest libdirac_encoder \
           libdirac_decoder encoder decoder util $(CPPUNITTESTS_DIR) \
--- origsrc/dirac-0.10.0/configure.ac	2008-05-12 05:49:33.000000000 -0500
+++ src/dirac-0.10.0/configure.ac	2008-06-15 03:40:58.812500000 -0500
@@ -16,7 +16,7 @@
 GENERIC_MICRO_VERSION=0
 
 #shared library versioning
-GENERIC_LIBRARY_VERSION=0:0:0
+GENERIC_LIBRARY_VERSION=$GENERIC_MINOR_VERSION:$GENERIC_MICRO_VERSION:0
 #                       | | |
 #                +------+ | +---+
 #                |        |     |
--- origsrc/dirac-0.10.0/dirac.pc.in	2008-05-04 21:14:06.000000000 -0500
+++ src/dirac-0.10.0/dirac.pc.in	2008-06-15 03:41:15.859375000 -0500
@@ -6,6 +6,6 @@
 Name: @GENERIC_LIBRARY_NAME@
 Description: The BBC Dirac Video Codec
 Version: @VERSION@
-Libs: -L${libdir} -l@GENERIC_LIBRARY_NAME@_encoder -l@GENERIC_LIBRARY_NAME@_decoder -lstdc++
+Libs: -L${libdir} -l@GENERIC_LIBRARY_NAME@_encoder -l@GENERIC_LIBRARY_NAME@_decoder
 Cflags: -I${includedir}/@GENERIC_LIBRARY_NAME@
 
